src/interpreter/ByteCode.h

```diff
@@ -827,6 +828,33 @@ class Const128 : public ByteCode {
     uint32_t m_value[4];
 };
 
+// dummy ByteCode for 3-input 1-output operation with offset
+class TernaryOperationOffset : public ByteCode {
```

Comment:
Based on the other common bases, the name should be `ByteCodeOffset4Value`. The class should be declared after `ByteCodeOffset2Value`.